<ui version="4.0" >
 <class>OSD_ConfigUI</class>
 <widget class="QWidget" name="OSD_ConfigUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>486</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="5" column="0" >
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QCheckBox" name="kcfg_UseOSD" >
     <property name="text" >
      <string>&amp;Enable On Screen Display</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>On &amp;Screen Display</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_OSDDrawShadow" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>&amp;Draw shadows</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text" >
         <string>OSD font:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_OSDFont</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KFontRequester" name="kcfg_OSDFont" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="kcfg_OSDOffsetX" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-2048</number>
        </property>
        <property name="maximum" >
         <number>2048</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="kcfg_OSDOffsetY" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-2048</number>
        </property>
        <property name="maximum" >
         <number>2048</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="kcfg_OSDAlignment" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox3" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Show &amp;OSD Message</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_OSDShowOwnNick" >
        <property name="text" >
         <string>&amp;If own nick appears in channel message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="kcfg_OSDShowChannel" >
        <property name="text" >
         <string>On an&amp;y channel message</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="kcfg_OSDShowQuery" >
        <property name="text" >
         <string>On &amp;query activity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="kcfg_OSDShowChannelEvent" >
        <property name="text" >
         <string>On &amp;Join/Part events</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="kcfg_OSDUseCustomColors" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Enable Custom Colors</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Te&amp;xt color:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_OSDTextColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>&amp;Background color:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_OSDBackgroundColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KColorButton" name="kcfg_OSDTextColor" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KColorButton" name="kcfg_OSDBackgroundColor" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QGroupBox" name="groupBox4" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>O&amp;ther Settings</string>
     </property>
     <layout class="QFormLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>&amp;Duration:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_OSDDuration</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel2_2" >
        <property name="text" >
         <string>S&amp;creen:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_OSDScreen</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="kcfg_OSDDuration" >
        <property name="suffix" >
         <string> ms</string>
        </property>
        <property name="minimum" >
         <number>500</number>
        </property>
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="singleStep" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="kcfg_OSDScreen" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_UseOSD</tabstop>
  <tabstop>kcfg_OSDDrawShadow</tabstop>
  <tabstop>kcfg_OSDFont</tabstop>
  <tabstop>kcfg_OSDOffsetX</tabstop>
  <tabstop>kcfg_OSDOffsetY</tabstop>
  <tabstop>kcfg_OSDAlignment</tabstop>
  <tabstop>kcfg_OSDShowOwnNick</tabstop>
  <tabstop>kcfg_OSDShowChannel</tabstop>
  <tabstop>kcfg_OSDShowQuery</tabstop>
  <tabstop>kcfg_OSDShowChannelEvent</tabstop>
  <tabstop>kcfg_OSDUseCustomColors</tabstop>
  <tabstop>kcfg_OSDTextColor</tabstop>
  <tabstop>kcfg_OSDBackgroundColor</tabstop>
  <tabstop>kcfg_OSDDuration</tabstop>
  <tabstop>kcfg_OSDScreen</tabstop>
  <tabstop>kcfg_OSDUseCustomColors</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_OSDDrawShadow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_OSDUseCustomColors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_OSDFont</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseOSD</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
